eegnb/cli/introprompt.py

```diff
@@ -93,6 +93,14 @@ def exp_prompt() -> str:
 
     return exp_selection
 
+def site_prompt() -> str:
+    print("\nPlease select which specific folder you would like to zip. Default 'local_ntcs' :")
```

Comment:
I think it's may be a little hard for a regular user to know what possible options exist.

Maybe mention this identifies the "experiment 'subfolder'"

Reply:
On seconds thought... a possible option here could be using the "experiment" value to check for the existing "subfolders / valid 'site'" options so the user knows which one to pick using os.listdir()

Reply:
Awesome. Thanks for adding @HVjay